packages/playground/src/weblets/tf_contracts_list.vue

```diff
@@ -187,6 +187,7 @@ async function onMount() {
   contracts.value = await getUserContracts(grid.value!);
   nodeStatus.value = await getNodeStatus(nodeIDs.value);
   loading.value = false;
+  selectedContracts.value = [];
```

Comment:
I think it is better to reset it before load the contracts, i mean line 185;
or at least before setting loading to false